static/js/containers/OrderSummaryPage.js

```diff
@@ -112,24 +112,33 @@ class OrderSummaryPage extends React.Component {
       coupons,
       location: { query }
     }  = this.props;
-    let courseRun, course, orderSummaryContent, coursePrice;
+    let orderSummaryContent, coursePrice;
     let courseKey = query.course_key;
-    [courseRun, course] = findCourseRun(dashboard.programs, run => run !== null && run.course_id === courseKey);
+    let [courseRun, course, program] = findCourseRun(
+      dashboard.programs, run => run !== null && run.course_id === courseKey
+    );
     if (prices.data) {
       coursePrice = prices.data.find(coursePrice => coursePrice.program_id === currentProgramEnrollment.id);
     }
 
-    if (course && courseRun && coursePrice) {
-      const [coupon, calculatedPrice] = calculatePrice(courseRun.id, course.id, coursePrice, coupons.coupons);
-      let discount = null;
-      if (calculatedPrice !== null && calculatedPrice !== undefined) {
-        discount = coursePrice.price - calculatedPrice;
+    if (program && course && courseRun && coursePrice && prices.data) {
```

Comment:
Could move line 121 into this section since you're checking for `prices.data` twice and it's not used outside this.